gepetuto/lint.py

```diff
@@ -8,21 +8,22 @@
 LOG = logging.getLogger("gepetuto.lint")
 
 
-def lint(tp_id: List[int], **kwargs):
+def lint(tp_id: List[int], file: List[str], **kwargs):
     """Lint python scripts."""
     LOG.info("linting tutorial sources.")
     for n in tp_id:
         LOG.debug(f"Looking for tp {n}")
-        lint_tp_folder(n)
+        folder = Path(f"tp{n}")
+        for python_file in folder.glob("*.py"):
+            lint_file(python_file)
+    for f in file:
+        lint_file(f)
```

Comment:
The idea here is to filter files. Here you are adding new ones.
I guess we should iterate over `file` **or** glob for `*.py` when it is empty,

Reply:
you mean that there may be file that aren't .py ones and we should filter them ?

Reply:
I was not clear, sorry for that.
I mean that if we have the following structure:
```
├── 1_example_notebook.ipynb
├── 2_other_notebook.ipynb
├── tp1
│   ├── cholesky.py
│   └── example_script.py
└── tp2
    ├── cholesky.py
    └── something.py
```

- with no tp_id and no file, we should find all 4 python files
- with tp_id=1 and no file, we should get the 2 python files in tp1
- with no tp_id and file=tp1/cholesky.py, we should get 1 python file in tp1
- with tp_id=1 and file=tp1/cholesky.py, we should get 1 python file in tp1
- with tp_id=2 and file=tp1/cholesky.py, we should get 0 python files

We'll also have to add `-F` / `--filter` to allow:
- with no tp_id and no file and filter=cholesky, we should get 2 python files, one in each tp
But that's for a separate future PR I think.

Reply:
no problem, understood